es-modules/src/components/todo-list.js

```diff
@@ -0,0 +1,50 @@
+import createElement from '../lib';
```

Comment:
The `lib` module doesn't actually contain a `default` export. However, `lib/index.js` does contain a _named_ export.